### PR TITLE
chore: changelog touchups

### PR DESCRIPTION
**Pull request by arboleya**
Left notes about the relevant changes.

Everything else is lintfixes.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Set branch name and changeset PR title
         run: |
-          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages for branch \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
+          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages - v`${{ env.BUILD_VERSION }}` @ \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
```
Comment (arboleya) [inline .github/workflows/release.yaml:54]: From:
 - ci(changesets): versioning packages for branch `master`

To:
 - ci(changesets): versioning packages - `v0.78.1` @ `master`

File: scripts/changeset/get-full-changelog.mts [hunk 2]
```diff
@@ -42,13 +45,14 @@ async function getChangelogInfo(octokit: Octokit, changeset: NewChangeset): Prom
...
   const isBreaking = title.includes(`${prType}!`);
 
-  const titleDescription = title.replace(/\w+\W+(.*)/, '$1'); // chore!: add something -> add something
-  const summary = titleDescription.charAt(0).toUpperCase() + titleDescription.slice(1);
+  const titleDescription = title.replace(/\w+\W+(.*)/, "$1"); // chore!: add something -> add something
+  const summary =
+    titleDescription.charAt(0).toUpperCase() + titleDescription.slice(1);
 
-  const markdown = `- ${prLink}, ${summary}, by ${user}`;
+  const markdown = `- ${prLink} - ${summary}, by ${user}`;
```
Comment (arboleya) [inline scripts/changeset/get-full-changelog.mts:55]: Replaced `,` by `-`.

From:
 - [#NNN](#), Pretty nice PR title, by @arboleya 

To:
 - [#NNN](#) - Pretty nice PR title, by @arboleya

File: scripts/changeset/update-changelog.mts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import * as core from "@actions/core";
 import * as github from "@actions/github";
 import { getInfo } from "@changesets/get-github-info";
+import { Octokit } from "@octokit/rest";
```
Comment (arboleya) [inline scripts/changeset/update-changelog.mts:4]: Not sure why this was skipped at first. 🤔

File: scripts/changeset/update-changelog.mts [hunk 4]
```diff
@@ -13,6 +14,34 @@ function sleep(time: number) {
   });
 }
 
+async function getChangesetPr(ghRepo: string, retried = false) {
+  const octokit = new Octokit();
```
Comment (arboleya) [inline scripts/changeset/update-changelog.mts:18]: Not sure why this was skipped at first. 🤔

File: scripts/changeset/get-full-changelog.mts [hunk 5]
```diff
@@ -144,9 +157,10 @@ export async function getFullChangelog(octokit: Octokit) {
   const breaking = listBreakingMd(changelogs);
   const nonBreaking = listNonBreakingMd(changelogs);
 
-  let content = `# RELEASE - ${process.env.RELEASE_TAG ?? 'TBD'}\n\n`;
-  content += breaking ? `# Breaking\n\n${breaking}` : '';
-  content += breaking && nonBreaking && '\n\n---\n\n';
+  let content = ``;
```
Comment (arboleya) [inline scripts/changeset/get-full-changelog.mts:160]: It became obsolete after the version was [added to the title](https://github.com/FuelLabs/fuels-ts/pull/1972/files#r1543512598).

File: .github/workflows/release.yaml [hunk 6]
```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Set branch name and changeset PR title
         run: |
-          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages for branch \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
+          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages - `v${{ env.BUILD_VERSION }}` @ \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
```
Comment (arboleya) [inline .github/workflows/release.yaml:54]: From:
 - ci(changesets): versioning packages for branch `master`

To:
 - ci(changesets): versioning packages - `v0.78.1` @ `master`

File: scripts/changeset/update-changelog.mts (hunk 4 above)
Comment (petertonysmith94) [reply scripts/changeset/update-changelog.mts:18]: https://github.com/FuelLabs/fuels-ts/pull/1972/files#diff-b5dbae1aaeb1da81c601f9a810383c55cd3fd847a69ef4567294bd82be355334R61

Do we not get the client via `@actions/github`?

Comment (arboleya) [reply scripts/changeset/update-changelog.mts]: Good catch! https://github.com/FuelLabs/fuels-ts/pull/1972/commits/3c9a1a008b6a723f2be50e01eeac80aa8a4c116d

File: scripts/changeset/update-changelog.mts (hunk 3 above)
Comment (arboleya) [reply scripts/changeset/update-changelog.mts]: https://github.com/FuelLabs/fuels-ts/commit/3c9a1a008b6a723f2be50e01eeac80aa8a4c116d

**Review by nedsalk — APPROVED** [review]
Nice!

**Review by petertonysmith94 — APPROVED** [review]

**Comment by petertonysmith94** [comment]
@arboleya your comments need resolving 😄

**Review by danielbate — APPROVED** [review]

**Pull request merged**